test/Elastica/Aggregation/GeoBoundsTest.php

```diff
@@ -8,7 +8,7 @@
 
 class GeoBoundsTest extends BaseAggregationTest
 {
-    protected function _getIndexForTest()
+    private function getIndexForTest()
```

Comment:
@Tobion You mean to do it across the whole project? Or you would do it step by step and only if the methods have to be changed anyways?

For the overall, I'm quite confident we will have some in 6.0. And for the change I don't worry too much about the users, I mostly worry about other libraries like FOSElastica where this is potentially going to break things.